bin/logify.pl

```diff
@@ -50,9 +82,70 @@
 sub logLineForDeclaration {
 	my $declaration = shift;
 	$declaration =~ m/^[+-]\s*\((.*?)\).*?/;
+
+	# if user only wants specific methods logified,
+	# either find those or exclude ones specified
+	if (defined $opt_exclude || defined $opt_filter) {
```

Comment:
Can you move this logic to a function? It'll make the code clearer. Something along the lines of `sub shouldLogDeclaration` that takes the `$declaration` at this point and returns a bool